src/main/webapp/help-timeout.html

```diff
@@ -0,0 +1,21 @@
+<!--
+    (c) Copyright 2020 Micro Focus or one of its affiliates.
+
+    Licensed under the MIT License (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    https://opensource.org/licenses/MIT
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+<div>
+    <p>
+        Apply a timeout in minutes (0 to 60) of waiting for SSC to finish processing artifacts to avoid hanging build.
```

Comment:
Let's confirm with Young the timeout max time. Because 60 min is might be too little. One of the SSC statuses that make polling repeated (from code above) is "Approve required". It means that somebody needs to accept the scan on SSC because of significant vulnerability count changes. So theoretically it is possible that somebody run the scan at the end of working day but will be able to approve it only next day after let's say 12 hours.

Reply:
I have set the maximum bound of timeout to 720 minutes and also updated the help message with the note for the empty case.